Gopkg.toml

```diff
@@ -67,10 +55,14 @@
   name = "gopkg.in/src-d/go-git.v4"
   revision = "v4.4.0"
 
-[[constraint]]
-  name = "github.com/google/go-containerregistry"
-  revision = "dc0a77702ddab8c3dc5630eca7c026d571c1c9b7"
-
 [[constraint]]
   name = "github.com/google/go-github"
   version = "15.0.0"
+
+[[constraint]]
+  name = "gopkg.in/AlecAivazis/survey.v1"
+  revision = "17861e192dc11fd2f5081df1932c94cce262fa1e"
+
+[[override]]
+  name = "github.com/docker/docker"
+  revision = "71cd53e4a197b303c6ba086bd584ffd67a884281"
```

Comment:
is 71cd53e4a197b303c6ba086bd584ffd67a884281 newer than 162ba6016def672690ee4a1f3978368853a1e149?